Fix incorrect peer dependencies and linter warnings

```
warning " > @thanos-wallet/dapp@1.1.3" has incorrect peer dependency "@taquito/taquito@^7.0.0 || ^7.0.0-beta".
warning " > formik-material-ui@3.0.1" has unmet peer dependency "tiny-warning@>=1.0.2".
warning " > react-compound-timer@1.2.0" has incorrect peer dependency "react@^16.8.6".
warning " > react-customizable-progressbar@1.0.2" has incorrect peer dependency "react@^0.14.0 || ^15.0.0 || ^16.0.0".
```
```
/home/runner/work/homebase-app/homebase-app/src/modules/common/Toolbar.tsx
  96:4  warning  Insert `,`  prettier/prettier

/home/runner/work/homebase-app/homebase-app/src/modules/explorer/pages/List.tsx
  83:5  warning  'error' is assigned a value but never used  @typescript-eslint/no-unused-vars

✖ 2 problems (0 errors, 2 warnings)
  0 errors and 1 warning potentially fixable with the `--fix` option.
```

--- a/src/modules/common/Toolbar.tsx
+++ b/src/modules/common/Toolbar.tsx
@@ -93,7 +93,7 @@ const AddressBarWrapper = styled(Grid)(({ theme }) => ({
   },
   [theme.breakpoints.down("xs")]: {
     marginLeft: -15,
-  }
+  },
 }));
 
 const custom = (theme: Theme, mode: "creator" | "explorer") => ({

--- a/src/modules/explorer/pages/List.tsx
+++ b/src/modules/explorer/pages/List.tsx
@@ -80,7 +80,6 @@ export const DAOsList: React.FC = () => {
   const [searchText, setSearchText] = useState("");
   const {
     data,
-    error,
     isLoading,
     hasNextPage,
     isFetchingNextPage,

